packages/framework/src/Component/UploadedFile/Config/Exception/InvalidUploadedFileConfigException.php

```diff
@@ -4,8 +4,8 @@
 
 namespace Shopsys\FrameworkBundle\Component\UploadedFile\Config\Exception;
 
-use Throwable;
+use Exception;
 
-interface UploadedFileConfigException extends Throwable
+abstract class InvalidUploadedFileConfigException extends Exception
```

Comment:
I'm not sure if we need to make any exceptions abstract.
Reason for it - if I need to create a new exception and place it properly into the exception hierarchy, I'm now put into thinking whether to make the parent exception abstract or not. 
For example, I have EntitiyIdentifierException and now I need to create `EntityIdentifierIsCompoundException extends EntityIdentifierException`. Should I rewrite everything and make `EntitiyIdentifierException` abstract?

Also, Exception is a class and now I'm extending a valid class into abstract class feels weird in a context of excetptions.

Reply:
ok, fair point, makes sense, I will remove the abstract declarations :+1:

Reply:
removed, rebased, force pushed